mrgnlabs/mrgn-ts
# feat(mfi-v2-ui): repay with collateral

- feat: static mint page UI
- feat: card / dialog variant updates, implemented on eco / mint
- feat: convertkit api integration
- chore: hide page header on desktop option
- wip: actionbox restructure
- feat: actionbox restructure
- fix: add lending wrapper to mobile
- chore: remove unused imports
- wip: lst actionbox
- wip: lst actionbox
- feat: lst actionbox
- feat: YBX actionbox
- chore: add sample script for bank params printing
- test comment out
- don't throw error when squadsX wallet creates account
- Update client.ts
- Update client.ts
- Update client.ts
- fix: update oracle label to read from str
- chore: add fixed fee to borrow apr / fix oracle tooltips since sdk change
- fix: add check that protocol fixed fee exists
- fix: only apply protocol fee to borrow rate
- fix: check protocol fee exists before comparison, add missing dep
- feat: default to mamas prio fee / jup image
- Update next.config.js
- feat: new tokens in actionbox
- chore: move new asset star to label
- feat new asset banner with ActionBox integration
- fix: log weight config
- feat: birdeye api
- chore: remove asset weight debugging
- fix: unnecessary useEffect dep
- chore: set default prio fee back to normal
- chore: increase birdeye cache ttl
- chore: remove birdeye cache debugs
- chore: upgrade posthog
- chore: use session storage to hide new asset banner
- fix: add referral link to accounts with null points data
- feat: add Hiemdall to ecosystem page
- feat: Add account authority transfer wrapper
- fix: Add doc on IDL generation
- fix: Run lint with prettier
- fix: Rerun IDL generation
- fix: Run prettier lint second time
- fix: Set feePayer to this.authority
- fix: Update README to reflect correct IDL generation
- fix: Push patched IDL (unformatted)
- fix: Push linted marginfi.json
- fix: Switch to setNewAccountAuthority()
- fix: Update to newAuthority field
- feat: wallet UI updates, accordion with tokens and notifications placeholder
- feat: wallet accordion for tokens / notifications
- feat: wallet notifications form UI, loading, success, error states
- feat: useConvertkit hook, implemented in WalletSettings for ybx / product update notifications
- feat: store / retrieve notifications settings
- feat: add points to wallet
- feat: add Hiemdall telegram bot link
- feat: wallet settings typoegraphy styling
- feat: add onramp button to wallet settings ui
- fix: account for usd soft limit == 0 i.e. disabled
- fix: update analytics link
- chore: release
- fix: add missing getter to the marginfi account wrapper
- chore: release
- fix: add missing group admin methods for account transfer WL
- chore: release
- chore: remove lfg emissions
- chore: mint page initial commit
- feat: mint page copy updates
- feat: integrations data
- feat: ybx partner dialog / form
- fix: lst mint dialog
- fix: add legacy ybx form env var until main merge
- feat: update mobile mint link
- fix: portfolio asset card rate ap check
- feat: update action complete to support LST minting
- chore: upgrade jupiter hook
- fix: jup v conflicts
- fix: rework LST amount in action complete dialog
- fix: try jsbi toString hack
- chore: copy update / button hover style
- feat: repay with collat wip checkpoint
